include/alpaka/pltf/PltfCudaRt.hpp

```diff
@@ -90,7 +90,9 @@ namespace alpaka
                     ALPAKA_DEBUG_FULL_LOG_SCOPE;
 
                     int iNumDevices(0);
-                    ALPAKA_CUDA_RT_CHECK(cudaGetDeviceCount(&iNumDevices));
+                    cudaError_t error = cudaGetDeviceCount(&iNumDevices);
+                    if(error != cudaSuccess)
+                        iNumDevices = 0;
```

Comment:
Note: This line is needed because it is not clear if CUDA is changing the parameter of `cudaGetDeviceCount` in the case where the return value is an error.